python/python/tests/test_async_client.py

```diff
@@ -7261,6 +7261,20 @@ async def test_copy_database(self, redis_client: GlideClient):
         finally:
             assert await redis_client.select(0) == OK
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_wait(self, redis_client: TGlideClient):
+        key = f"{{key}}-1{get_random_string(5)}"
+        value = get_random_string(5)
+        value2 = get_random_string(5)
+
+        assert await redis_client.set(key, value) == OK
+        assert await redis_client.wait(1, 1000) >= 0
```

Comment:
In Java we assert that the result is 0 if standalone, 1 if cluster, should we do the same here and below?